js/apps/admin-ui/maven-resources-community/theme/keycloak.v2/admin/messages/messages_es.properties

```diff
@@ -48,15 +48,15 @@ clientDescriptionHelp=Especifica la descripción del cliente. Por ejemplo, 'Mi c
 rolesPermissionsHint=Determina si se activan permisos detallados para gestionar este rol. Desactivarlo eliminará todos los permisos actuales que se hayan configurado.
 passwordPoliciesHelp.regexPattern=Requiere que la contraseña coincida con uno o más patrones de expresiones regulares de Java definidos.
 oAuthDPoP=Tokens de acceso con límite de DPoP de OAuth 2.0 habilitados
-invalidRealmName=El nombre del reino no puede contener caracteres especiales.
+invalidRealmID=ID del reino no puede contener caracteres especiales.
 validRedirectURIsHelp=Patrón de URI válido al que un navegador puede redirigir después de un inicio de sesión exitoso. Se permiten comodines simples como 'http://ejemplo.com/'. También se puede especificar una ruta relativa, como /mi/ruta/relativa/. Las rutas relativas son relativas a la URL raíz del cliente, o si no se especifica ninguna, se utiliza la URL raíz del servidor de autenticación. Para SAML, debes establecer patrones de URI válidos si dependes de la URL del servicio consumidor incrustada en la solicitud de inicio de sesión.
 realmNameTitle=Reino {{nombre}}
 subjectNameId=NombreID del sujeto
 credentialsList=Lista de credenciales
 usermodel.clientRoleMapping.clientId.label=ID de cliente
 clientId=ID de cliente
 serviceProviderEntityId=ID de entidad del proveedor de servicios
```

Comment:
Some of these changes have nothing to do with the PR. But they fix some encoding issues.

If not wanted, please let me know so that I rollback them.